supabase/functions/process-receipt/index.ts

```
@@ -52,35 +52,48 @@ function parsePurchaseDate(dateString: string | undefined): string | null {
+
 // Helper function to extract date from filename
 function extractDateFromFilename(filename: string | null): string | null {
   if (!filename) return null;
-  
+
   // Remove file extension
   const nameWithoutExt = filename.replace(/\.[^/.]+$/, '');
-  
+
   // Pattern 1: YYYY-MM-DD or YYYY_MM_DD or YYYY.MM.DD
   const isoPattern = /(\d{4})[-_.\/](\d{1,2})[-_.\/](\d{1,2})/;
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)